include/realtime_tools/realtime_publisher.h

```diff
@@ -227,6 +227,10 @@ class RealtimePublisher : boost::noncopyable
   int turn_;  // Who's turn is it to use msg_?
 };
 
+#include <memory>
+template <class Msg>
+using RealtimePublisherSharedPtr = std::shared_ptr<RealtimePublisher<Msg> >;
+
```

Comment:
This could be moved into the class as well.
Not sure which style to follow here..

Reply:
Here, please correct if I'm wrong but I think this is how the alternatives would look like:

Eg:
`boost::shared_ptr<realtime_tools::RealtimePublisher<sensor_msgs::JointState> > realtime_pub_;`
to 
`realtime_tools::RealtimePublisher<sensor_msgs::JointState>::SharedPtr realtime_pub_;`
or
`realtime_tools::RealtimePublisherSharedPtr<sensor_msgs::JointState> realtime_pub_;`

I'd personally opt for less namespacing.

Reply:
I was just curious if there are any good reason to prefer one over the other.

> I'd personally opt for less namespacing.

:+1: